### PR TITLE
Refactor Privacy.react.js, convert to functional component

### LINKED ISSUE #1695: Refactor Privacy.react.js, make it functional component
**Actual Behaviour**

Make NotFound consistent with codebase, it is only a presentational component. Make it functional and clean up the codebase by removing unnecessary states.

**Expected Behaviour**

Refactor NotFound.react.js

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1695 
Changes: 

- Ordered import statements
 - constant declared before components to make it consistent
 - Removed redundant state, made it functional component.
 - Remove unwanted class methods.

Demo Link: https://pr-1695-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
Please Ignore codacy.

File: src/components/Privacy/Privacy.react.js [hunk 1]
```diff
@@ -1,344 +1,308 @@
...
-      showOptions: true,
-      anchorEl: event.currentTarget,
-    });
-  };
-
-  _onReady(event) {
-    // access to player in all event handlers via event.target
-    event.target.pauseVideo();
-  }
+import $ from 'jquery';
```
Comment (AakashMallik) [inline src/components/Privacy/Privacy.react.js]: We have been grouping node modules together and app components together, so I guess $ should be shifter just after ```import PropTypes```

**Comment by AakashMallik** [comment]
@fragm3 Squash the commits too.

**Comment by fragm3** [comment]
@AakashMallik done

**Comment by AakashMallik** [comment]
> @AakashMallik done

:sweat_smile: I don't see the changes, plus I see two commits. Squashing should have reduced them to just one.

**Comment by fragm3** [comment]
@anshumanv I have already squashed the commits, after squashing it shows update branch on website, I did that.

**Comment by anshumanv** [comment]
Yeah, it's inevitable I guess, thanks! 😄

**Pull request merged**